product-service/src/handlers/addProduct.js

```diff
@@ -0,0 +1,14 @@
+import { addProductToDb } from '../services/addProductToDb/addProductToDb';
+import { buildHttpResponse } from '../utils/buildHttpResponse';
+
+export const addProduct = async (event, context) => {
+    const { title = '', description = '', price = 0, image = '' } = JSON.parse(event.body);
+    console.log(JSON.parse(event.body))
+    if (!title || !description || !price || !image) return buildHttpResponse(400, { errorMessage: 'Product data is invalid' });
```

Comment:
According to model only title is required. By the way, don't you want to use type validation too?